src/App.js

```diff
@@ -12,6 +14,7 @@ function App() {
       <div>
         <main className='main-content'>
           <Route exact path={ROUTES.LANDING} component={Landing} />
```

Comment:
Same path for two routes

Reply:
Login component could be added to the Landing page instead of creating own page for it